src/common/args.cpp

```diff
@@ -700,10 +765,11 @@ std::string HelpMessageGroup(const std::string &message) {
     return std::string(message) + std::string("\n\n");
 }
 
-std::string HelpMessageOpt(const std::string &option, const std::string &message) {
-    return std::string(optIndent,' ') + std::string(option) +
-           std::string("\n") + std::string(msgIndent,' ') +
-           FormatParagraph(message, screenWidth - msgIndent, msgIndent) +
+std::string HelpMessageOpt(const std::string& option, const std::string& help_param, const std::string &message, int indent)
+{
+    return std::string(optIndent+indent,' ') + option + help_param +
+           std::string("\n") + std::string(msgIndent+indent,' ') +
+           FormatParagraph(message, screenWidth - msgIndent - indent, msgIndent + indent) +
            std::string("\n\n");
```

Comment:
```suggestion
    return strprintf("%*s%s%s\n%*s%s\n\n",
                 optIndent + indent, "", option, help_param,
                 msgIndent + indent, "",
                 FormatParagraph(message, screenWidth - msgIndent - indent, msgIndent + indent));
```

Reply:
`FormatParagraph()` only indenting when it sees a `\n` seems plausible for dealing with an indented continuation (ie, `"Hello " + FormatParagraph("world!\n", 79, 4)`) and the current behaviour is tested, so I'd rather not touch that.